### PR TITLE
feat: showing edit project state in task list

### PR DESCRIPTION
**Pull request by gurjmatharu**
#439

### CODE REVIEW

File: app/tests/unit/pages/project-revision/[projectRevision]/index.test.tsx [hunk 1]
```diff
@@ -101,14 +104,34 @@ describe("The Create Project page", () => {
...
+  //     within(
+  //       screen.getByRole("navigation", { name: "side navigation" })
+  //     ).getByText(/( optional )/i)
+  //   ).toBeInTheDocument();
+
+  //   expect(
+  //     screen.getByText(/review and submit information/i).closest("li")
+  //   ).toHaveAttribute("aria-current", "step");
+  // });
+
```
Comment (matthieu-foucault) [inline app/tests/unit/pages/project-revision/[projectRevision]/index.test.tsx:134]: do you want to remove this?

File: app/cypress/integration/cif/project-revision/index.spec.js [hunk 2]
```diff
@@ -170,7 +170,7 @@ describe("the new project page", () => {
     cy.contains("Changes saved.");
```
Comment (matthieu-foucault) [inline app/cypress/integration/cif/project-revision/index.spec.js:170]:
```suggestion
    cy.contains("Changes saved.");
    cy.get("body").happoScreenshot({
      component: "Project Overview Form",
      variant: "editing",
    });
```

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

File: app/tests/unit/pages/project-revision/[projectRevision]/index.test.tsx (hunk 1 above)
Comment (gurjmatharu) [reply app/tests/unit/pages/project-revision/[projectRevision]/index.test.tsx:134]: nope - this was a mistake, fixing.

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

File: app/components/TaskList.tsx [hunk 3]
```diff
@@ -51,28 +60,28 @@ const TaskList: React.FC<Props> = ({ projectRevision }) => {
           </ul>
```
[the remark this answers is not in the document]
Comment (gurjmatharu) [reply app/components/TaskList.tsx:60]: updated.

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**